### ISSUE TITLE
GO term specificity proposal 2  (gene product specific regulation via catabolism)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#12417]
Remove "gene product specific" regulation via catabolism.

This term, and its descendants
GO:0008054 negative regulation of cyclin-dependent protein serine/threonine kinase by cyclin degradation

cyclin is degraded by the Anaphase promoting complex. This does regulate Cyclin dependent-kinase but this seems a pretty convoluted way to represent this.

We would use
GO:0031145
anaphase-promoting complex-dependent proteasomal ubiquitin-dependent protein catabolic process
has_input (cyclin)
(note this term is now represented more succinctly as 
"anaphase-promoting complex catabolic process"

this is clearer, and more informative because it captures the specific pathway, and the substrate

**Comment by ValWood** [comment]
@pgaudet @thomaspd @dosumis

**Comment by dosumis** [comment]
But this is easy to read and understand: "negative regulation of cyclin-dependent protein serine/threonine kinase by cyclin degradation"

While this is pretty hard to parse on first or even second reading - especially if you don't already understand the biology enough to know how to chunk it. (Best to avoid heavily stacked noun phrases like this):
"anaphase-promoting complex-dependent proteasomal ubiquitin-dependent protein catabolic process"

There's also nothing wrong with the logic of the first phrase - regulation may be indirect. 

Here's an attempt to break it down to something like a LEGO model using annotation extensions (A LEGO model would be clearer and more accurate though.

'proteasomal ubiquitin-dependent protein catabolic process' has_agent anaphase-promoting complex, has_substrate 'cyclin', regulates cyclin-dependent protein serine/threonine kinase activity.

BTW 'proteasomal ubiquitin-dependent protein catabolic process' is a horrible name.  Wouldn't it be better to have 'ubiquitin-depend protein degredation by proteasome' ?

**Comment by dosumis** [comment]
Some thoughts following discussion in editor's meeting:

* Scratch the last suggestion. Given that the existing term is used to annotate APC and it's components, it makes no sense to an extension to with has_agent APC. But then again it also seems odd to have a term which names APC, specifically for the purpose of annotating APC and its components.  Surely you could just annotate APC components to any term that starts 'APC dependent...' without knowing any biology at all.  Isn't it better to annotate the APC complex using MF and BP terms that make clear what it does, and then propagate to genes using the contributes_to qualifier?

* I can't make sense of the use case here without a nice textual description of the biology and then maybe a LEGO model.  Here's my quick attempt:

https://en.wikipedia.org/wiki/Anaphase-promoting_complex:

"Anaphase-Promoting Complex (also called the cyclosome or APC/C) is an E3 ubiquitin ligase that marks target cell cycle proteins for degradation by the 26S proteasome."

Ubiquitination regulates degradation of some target - regulating the overall level of activity of that target in a cell.  In this case the target is some cyclin and the result of regulating levels of this cyclin is regulation of the overall level of cyclin dependent kinase activity 

Surely the primary annotations should be to 

MF: E3 ubiquitin ligase activity
BP: protein ubiquitination leading to degradation # Made this up but it seems reasonable

The BO could then be extended:

protein ubiquitination leading to degradation
 * has_input some cyclin
 * regulates some cyclin dependent kinase activity

LEGO is clearly much better suited to capturing the full story, but this gets us some of the way there and uses terms that non-specialists can follow.  It should also be possible to make this fully compatible with a LEGO representation via the consistent use of patterns in the ontology, extensions and LEGO.

**Comment by tberardini** [comment]
One proposal that looked like it might be favored was to merge GO:0008054 (the 'cyclin degradation' term)  into GO:0031145 (the 'APC-dependent catabolism' term). It was proposed to then eliminate the current primary term name of 8054, 'negative regulation of cyclin-dependent protein serine/threonine kinase by cyclin degradation' as a synonym and retain all the 'cyclin degradation'-related synonyms.

**Comment by ValWood** [comment]
What did we decide... I still vote to get rid of the gene product specific catabolism terms, there are much cleaner ways to make these annotations (the suggested merge would work fine). Does this  need further discussion?

**Comment by tberardini** [comment]
I don't recall a discussion or what was decided beyond setting this issue to 'pending'.  @paolaroncaglia , @ukemi , @dosumis , @mcourtot  - can someone remember more?

**Comment by paolaroncaglia** [comment]
Sorry, can't remember more :-(

**Comment by ukemi** [comment]
Me neither.

**Comment by tberardini** [comment]
Here's what I suggest to close this issue:

- [ ] merge _GO:0008054 negative regulation of cyclin-dependent protein serine/threonine kinase by cyclin degradation_ into _GO:0031145 anaphase-promoting complex-dependent catabolic process_
- [ ]  merge the children terms [pos/neg/regulation of cyclin catabolic process] into [pos/neg/regulation of cellular protein catabolic process]

**Comment by ValWood** [comment]
I think that is what the proposed solution was....
I think I was supposed to update the tickets after the meeting. I found the older on my desktop, so there was clearly something I was still supposed to do here, but hadn't...

**Comment by tberardini** [comment]
Implemented changes above.  Closing.
